CMakeLists.txt

```diff
@@ -166,15 +166,17 @@ else()
       if (STRIP STREQUAL "STRIP-NOTFOUND")
           message(FATAL_ERROR "strip not found")
       endif()
-
-    else (CMAKE_SYSTEM_NAME STREQUAL Darwin)
-
+    elseif (CMAKE_SYSTEM_NAME STREQUAL Linux)
       # Ensure that objcopy is present
-      find_program(OBJCOPY objcopy)
+      if(DEFINED ENV{CROSSCOMPILE})
+        find_program(OBJCOPY arm-linux-gnueabihf-objcopy)
+      else()
+        find_program(OBJCOPY objcopy)
+      endif()
       if (OBJCOPY STREQUAL "OBJCOPY-NOTFOUND")
           message(FATAL_ERROR "objcopy not found")
       endif()
-    endif (CMAKE_SYSTEM_NAME STREQUAL Darwin)
+    endif ()
```

Comment:
I think it should still handle Mac OS X which is lost here by the removal of the if condition. Also it should handle both cross compilation for arm and arm64.